src/stories/atoms/MovieCard.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "../assets/styles/MovieCard.scss";
+
+const MovieCard = ({ imageUrl, name, year, genreList }) => {
+
+    const createListOfGenres = () => {
```

Comment:
Since we don't have requirements on how to handle cases if a movie isn't attached to some genre, we can keep this logic. But we're using the same function in the `MovieCard` component, let`s move it to a separated reusable utility.